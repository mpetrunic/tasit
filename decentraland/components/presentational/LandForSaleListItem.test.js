import React from "react";
import { shallow } from "enzyme";
import LandForSaleListItem from "./LandForSaleListItem";

describe("LandForSaleListItem", () => {
  it("renders the component", async () => {
    const landForSale = {
      id: 123,
      priceMana: 0,
      priceUSD: 0,
      asset: {
        id: -1,
        name: "Not found",
        img: "https://decentraland.org/images/logo-65f7b27caf.png",
      },
    };
    const onPress = () => {};
    expect(
      shallow(
        <LandForSaleListItem landForSale={landForSale} onPress={onPress} />
      )
    ).toMatchSnapshot();
  });
});
