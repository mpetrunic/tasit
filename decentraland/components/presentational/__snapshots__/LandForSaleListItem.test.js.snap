// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandForSaleListItem renders the component 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  delayPressOut={100}
  onPress={[Function]}
  underlayColor="black"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginBottom": 13.34,
        "paddingBottom": 40.019999999999996,
        "paddingLeft": 22.5,
        "paddingRight": 22.5,
        "paddingTop": 40.019999999999996,
      }
    }
  >
    <LandForSale
      landForSale={
        Object {
          "asset": Object {
            "id": -1,
            "img": "https://decentraland.org/images/logo-65f7b27caf.png",
            "name": "Not found",
          },
          "id": 123,
          "priceMana": 0,
          "priceUSD": 0,
        }
      }
    />
  </Component>
</TouchableHighlight>
`;
